### DISCUSSION TITLE
[SOLVED] Documentation gone. Why?

### DISCUSSION THREAD

**Discussion by Starboy-Xo** [martinrotter/rssguard#1113] [Q&A] [answered]
So, documentation has gone. why is it so?

**Comment by martinrotter** [comment, marked as answer]
Hey. Yes, I removed the documentation and will be moving it to ReadTheDocs and greatly enhancing it. I will fix all links to documentation when the process will be done, perhaps tomorrow.

Here is very raw and incomplete and WIP new docs.

https://rssguard.readthedocs.io/en/latest/

Link to old docs.

https://github.com/martinrotter/rssguard/blob/4.5.0/resources/docs/Documentation.md

**Reply by Starboy-Xo** [reply]
Thanks.
